Makefile

```diff
@@ -3,7 +3,17 @@
 # To re-generate a bundle for another specific version without changing the standard setup, you can:
 # - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
 # - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
-VERSION ?= $(or $(shell git describe --abbrev=0 --tags 2>/dev/null),$(GIT_HEAD_COMMIT))
+# VERSION ?= $(or $(shell git describe --abbrev=0 --tags 2>/dev/null),$(GIT_HEAD_COMMIT))
+
+ifndef OVERRIDE_KAMAJI_VERSION
+	VERSION = $(shell ./get-label.bash)
+else
+	ifneq ($(strip $(OVERRIDE_KAMAJI_VERSION)),)
+		VERSION ?= $(OVERRIDE_KAMAJI_VERSION)
+	else
+		VERSION = $(shell ./get-label.bash)
+	endif
```

Comment:
I like this pattern for versioning - that we also use in Luigi I think. Get the tag from GitHub if it exists - else use the commit SHA ID. We should document this in the README.md